app/translations/dutch.js

```diff
@@ -0,0 +1,396 @@
+  'errors.request.fractional': `Je kunt niet een gedeelte van {asset} opvragen.`,
+  'errors.request.validDecimals': `Je kunt {asset} tot maximaal {validDecimals, number} aanvragen.`,
+  'errors.request.max': `Je kunt niet meer dan 100.000.000 {asset} aanvragen.`,
+  'errors.request.min': `Je kunt niet 0 {asset} aanvragen.`,
+  'errors.network.general': 'Oops! Iets ging verkeerd.',
+  'errors.encrypt.valid': 'De private sleutel is niet geldig.',
+
+  'errors.send.balance': `Je hebt niet genoeg in je wallet om {total} {asset} te verzenden.`,
+  'errors.send.network': 'Een netwerk probleem is opgetreden.',
+  'errors.send.number': 'Aantal moet nummeriek zijn.',
+  'errors.send.fraction': 'Je knt geen gedeelte van een NEO verzenden.',
+  'errors.send.negative': `Je unt geen negatieve aantallen {asset} verzenden`,
+  'errors.send.zero': `Je kunt geen 0 {asset} verzenden.`,
+  'errors.send.decimal': `Je kunt {asset} tot maximaal {decimalCount, number} verzenden.`,
+  'errors.send.invalidAddress': 'Je moet een geldig NEO adres invoeren..',
+  'errors.send.blackListed':
+    'Adres is geblacklist, dit is een bekend phising adres.',
+}
+
+const NOTIFICATIONS = {
+  'notifications.success.receivedBlockchainInfo':
+    'Laaste blockchain informatie opgevraagd.',
+  'notifications.success.accountSaved': 'Account opgeslagen!',
+  'notifications.success.updatedWalletName': 'Wallet naam opgeslagen..',
+  'notifications.failure.blockchainInfoFailure':
+    'Het is niet gelukt om blockchain informatie op te halen.',
+}
+
+const AUTH = {
+  authLogin: 'Inloggen',
+  authSaved: 'OPGESLAGEN',
+  authPrivate: 'PRIVATE KEY',
+  authEncrypted: 'VERSLEUTELD',
+  authWatch: 'ALLEEN LEZEN',
+  authLedger: 'LEDGER',
+  authCreateWallet: 'Maak wallet',
+  authImportWallet: 'Importeer wallet',
+  authScanQRButton: 'Scan QR-Code',
+  authLoginButton: 'Inloggen',
+  authLedgerFirstStep: 'Verbind en ontgrenden je Ledger-apparaat',
+  authLedgerSecondStep: 'Navigeer naar de NEO-app op je apparaat',
+  authLedgerAddressLabel: 'PUBLIEK ADRES',
+}
+
+const WALLET_CREATION = {
+  createANewWallet: 'Maak nieuwe wallet',
+  walletCreationInstruction: 'Voer details in',
+  walletCreationWalletNameLabel: 'WALLET NAAM',
+  walletCreationWalletNamePlaceholder: 'Wallet naam',
+  walletCreationWalletPasswordLabel: 'Wachtwoord',
+  walletCreationWalletPasswordPlaceholder: 'Wachtwoord',
+  walletCreationWalletPasswordConfirmLabel: 'Herhaal wachtwoord',
+  walletCreationWalletPasswordConfirmPlaceholder: 'Wachtwoord nogmaals',
+  walletCreationButton: 'Maak wallet',
+  walletCreatedHeader: 'Wallet aangemaakt!',
+  walletImportedHeader: 'Wallet aangemaakt!',
+  walletCreatedDisclaimer:
+    '<b>Bewaar deze details!</b> Als je deze inloggegevens kwijtraakt, heb je geen toegang meer tot je assets.',
+  privateKeyLabel: 'PRIVATE KEY',
+  encryptedKeyLabel: 'VERSLEUTELDE KEY',
+  addressLabel: 'PUBLIEK ADRES',
+  splitKeyLabel: 'GESPLITSTE SLEUTEL',
```

Comment:
inconsistent translation; either use no translation ("private key") everywhere, or full translation "privé sleutel". This is a mix of sometimes partially translated, sometimes not at all.

Reply:
I agree, I've changed as follows. Public key and private keys do not have descent translations in Dutch. So I've used public key and private key as terms: 'names' so to say. The encrypted key is now a 'Beveiligde Sleutel'.